rke/k8s_rke_system_images.go

```diff
@@ -6261,6 +6261,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.6.4
```

Comment:
2.6.4 will likely go out with Feb 2022 patches, so this should be `// Enabled out of band post v2.6.3` (same for other versions we introduce here).